src/styles/index.scss

```diff
@@ -1,3 +1,159 @@
-body {
+$background-size-h: 408px;
+$background-size-w: 200px;
+$blue-accent:  #00acdc;
+$button-h: 40px;
+$button-w: 166px;
+$border-radius: 5px;
+$star-size: 16px;
+$main-color:#060b35;
+$side-text-color: #616070;
+$light-color: #fff;
+$border-color: #f3f3f3;
+
+*,
+*::before,
+*::after {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
```

Comment:
you've already reset those styles, please remove them, decide where do you want them to go 😉